src/core/utils-dom.ts

```diff
@@ -7,3 +7,23 @@ import * as React from 'react';
 import { act as reactDomAct } from 'react-dom/test-utils';
 
 export const act = ('act' in React ? React.act : reactDomAct) as typeof reactDomAct;
+
+// Copied from @testing-library/dom/dist/events.js
```

Comment:
I'm not sure this is sufficient attribution for the usage. We might need to include the license type as well as a link to the original source code.

Or maybe something similar to what we have here: https://github.com/cloudscape-design/components/blob/main/THIRD-PARTY-LICENSES

Reply:
Do we have any other examples in the components repo of usages of just a single function being copied in?